imagersite/imager_profile/forms.py

```diff
@@ -1,9 +1,30 @@
 from django import forms
 from .models import ImagerProfile
-# from django.contrib.auth.models import User
 
 
 class ProfileSettingsForm(forms.ModelForm):
+    first_name = forms.CharField(label='First Name', max_length=36)
+    last_name = forms.CharField(label='Last Name', max_length=36)
+    email = forms.EmailField(label='Email')
```

Comment:
good solution.  Add fields for those fields that are on the User, but not the Profile.